lib/modules/manager/npm/extract/index.ts

```diff
@@ -138,6 +143,19 @@ export async function extractPackageFile(
   const yarnrcYmlFileName = getSiblingFileName(fileName, '.yarnrc.yml');
   const yarnZeroInstall = await isZeroInstall(yarnrcYmlFileName);
 
+  let yarnConfig: YarnConfig | null;
+  const repoYarnrcYml = await readLocalFile(yarnrcYmlFileName, 'utf8');
+  if (is.string(repoYarnrcYml)) {
+    if (is.string(config.npmrc) && !config.npmrcMerge) {
+      logger.debug(
+        { yarnrcYmlFileName },
+        'Repo .yarnrc.yml file is ignored due to config.npmrc with config.npmrcMerge=false'
+      );
+    } else {
```

Comment:
Does this makes any sense? yarn v2+ fully ignores `npmrc` 🤔

Reply:
Good point. Maybe we should just get rid of this check. I don't think the new functionality will clash with the existing `npmrc` handling.